libs/service-portal/health/src/screens/DentistRegistration/DentistRegistration.tsx

```
@@ -219,19 +193,69 @@ export const DentistRegistration = () => {
+                              )} ${selectedDentist?.name}`}
+                              buttons={[
+                                {
+                                  id: 'RegisterModalAccept',
+                                  type: 'primary' as const,
+                                  text: formatMessage(
+                                    messages.healthRegisterModalAccept,
+                                  ),
+                                  onClick: () => {
+                                    setErrorTransfering(false)
+                                    setModalVisible(false)
+                                    if (selectedDentist && selectedDentist.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)